thanos/README.md

```diff
@@ -2,29 +2,63 @@
 title: Thanos
 ---
 
-# Information
+Learn more about the **Thanos** Plugin. Use it to enable extended metrics retention and querying across Prometheus servers and Greenhouse clusters. 
+
+The main terminologies used in this document can be found in [core-concepts](https://cloudoperators.github.io/greenhouse/docs/getting-started/core-concepts).
+
+## Overview
+
+Observability is often required for operation and automation of service offerings. To get the insights provided by an application and the container runtime environment, you need telemetry data in the form of _metrics_ or _logs_ sent to backends such as _Prometheus_ or _OpenSearch_. With the **Thanos** plugin you can scale the _metric_ part of the observability stack horizontally by hooking into the Prometheus provided by the **kube-monitoring** plugin and optionally save your data in the object storage.
```

Comment:
```suggestion
Observability is required for monitoring the lifecycle of service offerings. To get the insights provided by an application and the container runtime environment, you need telemetry data in the form of _metrics_ or _logs_ which are made accessible through backends such as _Prometheus_ or _OpenSearch_.
With the **Thanos** plugin you can enhance the _metric_ part of the observability stack to scale horizontally and leverage object storage for a long retention setup.
```

What explicitly do you mean with `container runtime environment`?

To me this is/was somehow hard to read and grasp. I rewrote it, not sure if this is understandable - or even needed. The following section (`In general, Thanos is...`)is more to the point, while this is too generic. I would wipe it entirely. Similar wording could go in an overall metrics architecture document.

Furthermore, I don't see why logs w/ need to be mentioned here, this also would rather be part of an overarching document.

Reply:
Kubernetes. 🤣 
> What explicitly do you mean with container runtime environment?


You're right, I admit. We don't have space for such a general document. That's why it's included in every plugin at the moment. We can also remove it and I will find a place for it.

> To me this is/was somehow hard to read and grasp. I rewrote it, not sure if this is understandable - or even needed. The following section (In general, Thanos is...)is more to the point, while this is too generic. I would wipe it entirely. Similar wording could go in an overall metrics architecture document.
> Furthermore, I don't see why logs w/ need to be mentioned here, this also would rather be part of an overarching document.